app/src/main/java/org/simple/clinic/settings/changelanguage/ChangeLanguageEvent.kt

```diff
@@ -4,10 +4,12 @@ import org.simple.clinic.settings.Language
 
 sealed class ChangeLanguageEvent
 
-data class CurrentSelectedLanguageLoadedEvent(val language: Language) : ChangeLanguageEvent()
+data class CurrentLanguageLoadedEvent(val language: Language) : ChangeLanguageEvent()
 
 data class SupportedLanguagesLoadedEvent(val languages: List<Language>) : ChangeLanguageEvent()
 
-data class SelectLanguageEvent(val newLanguage: Language): ChangeLanguageEvent()
+data class SelectLanguageEvent(val newLanguage: Language) : ChangeLanguageEvent()
```

Comment:
`CurrentLanguageChangedEvent` stands out from `UpdateCurrentLanguageEffect` and `CurrentLanguageChangedEvent`. Can we make these more alike?

Reply:
Discussed and explained offline.